src/elements/molecules/Expand/Expand.jsx

```diff
@@ -69,13 +72,15 @@ export class Expand extends React.Component {
       className
     ]);
 
+    const expanded = defaultState !== 'closed';
+
     return (
       <Tag
         className={classStack}
         {...attrs}
       >
-        <Heading level={level} className="expand__trigger">{title}</Heading>
-        <section className="expand__target">
+        <Heading tagName="button" level={level} className="expand__trigger" aria-expanded={expanded} id={id}>{title}</Heading>
+        <section className="expand__target" aria-hidden={!expanded} aria-labelledBy={id}>
```

Comment:
aria-labeledBy needs to be moved to the `<Tag` in render, and lets add aria-describedBy too. Both of these should look like:

```javascript
<Tag
  className={classStack}
  aria-labelledBy={`${id}ExpandTitle`}
  aria-describedBy={`${id}ExpandDesc`}
  {...attrs}
```
Heading and section should just have ids that equal the value of our labelledby and describedBy attributes
```javascript
<Heading tagName="button" level={level} className="expand__trigger" aria-expanded={expanded} id={`${id}ExpandTitle`}>{title}</Heading>
<section className="expand__target" aria-hidden={!expanded} id={`${id}ExpandDesc`}>
  {children}
</section>
```